client/state/partner-portal/licenses/hooks/use-products-query.ts

```diff
@@ -49,11 +50,14 @@ function queryProducts(): Promise< APIProductFamily[] > {
 export default function useProductsQuery(): UseQueryResult< APIProductFamilyProduct[], unknown > {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
+	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
 
 	const query = useQuery( {
 		queryKey: [ 'partner-portal', 'licenses', 'products' ],
 		queryFn: queryProducts,
 		select: selectAlphabeticallySortedProductOptions,
+		enabled: isPartnerOAuthTokenLoaded,
+		refetchOnWindowFocus: false,
```

Comment:
the `useProductsQuery` was retrieving products from the API call on each windows focus, which means we were getting the same products each time it got the focus, it was annoying to see the multiple calls nonsense.

Reply:
That was happening with the Email Previewer, too...one of the little things I fixed. Really annoying.